tests/introspection/ApiSelectorTest.cs

```diff
@@ -840,6 +840,35 @@ protected virtual bool Skip (Type type, string selectorName)
 				if (selectorName == "waitUntilExit")
 					return true;
 				break;
+			case "SHCustomCatalog":
+				// ShazamKit has some issues running on simulator
+				switch (selectorName) {
+				case "new":
+					return true;
+				}
+				break;
```

Comment:
If you look at others, we have code that will return true only if in the simulator, you should do the same.

Reply:
Ah good call, thank you!